### DISCUSSION TITLE
Cannot read properties of undefined (reading 'gasLimit')

### DISCUSSION THREAD

**Discussion by nakshatrasinghh0x** [wevm/wagmi#1096] [Q&A]
Please help me with this, I am signing a transaction with two arguments in write (address, metadataURL). 

WAGMI version: ^0.6.8

**Comment by tmm** [comment]
Can you post a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)?

**Comment by nakshatrasinghh0x** [comment]
Hey @tmm, the issue was resolved, i just added mode type as reckless and it worked, cheers
